### ISSUE TITLE
Travis issues logged but build passing

### ISSUE THREAD

**Issue by ba66e77** [acquia/blt#3084]
In reviewing #3082 , the SQLSTATE message below was observed, even though Travis build still passed.  

```
In Connection.php line 686:
                                                                               
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'drupal.key_value  
  ' doesn't exist: DELETE FROM {key_value}                                     
  WHERE (name IN (:db_condition_placeholder_0)) AND (collection = :db_conditi  
  on_placeholder_1); Array                                                     
  (                                                                            
      [:db_condition_placeholder_0] => drupal_css_cache_files                  
      [:db_condition_placeholder_1] => state                                   
  )                                                                            
                                                                               
In Statement.php line 59:
                                                                               
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'drupal.key_value  
  ' doesn't exist                                                              
``` 

Looking into that one, @lcatlett determined the issue was not introducted in #3082 and also noticed a co-occurring issue:
                    
```
Warning: include(../../scripts/phpunit/bootstrap.php): failed to open stream: No such file or directory in /home/travis/build/acquia/blt/tests/phpunit/bootstrap.php on line 8
Warning: include(): Failed opening '../../scripts/phpunit/bootstrap.php' for inclusion (include_path='.:/home/travis/.phpenv/versions/7.1.11/share/pear') in /home/travis/build/acquia/blt/tests/phpunit/bootstrap.php on line 8
PHP Warning:  include(../../scripts/phpunit/bootstrap.php): failed to open stream: No such file or directory in /home/travis/build/acquia/blt/tests/phpunit/bootstrap.php on line 8
PHP Warning:  include(): Failed opening '../../scripts/phpunit/bootstrap.php' for inclusion (include_path='.:/home/travis/.phpenv/versions/7.1.11/share/pear') in /home/travis/build/acquia/blt/tests/phpunit/bootstrap.php on line 8
Bootstrapping BLT testing framework...
To prevent recreation of sandbox master on each bootstrap, set BLT_RECREATE_SANDBOX_MASTER=0
Creating master sandbox in /tmp/blt-sandbox-master...
Loading composer repositories with package information
```

This is occurring in both 9.x and 9.2.x versions. I've tracked the issue back as far as the 9.1.1 release, but have not yet determined the PR withing that release where it first arose.

**Comment by lcatlett** [comment]
@ba66e77 I was taking a look at the `SQLSTATE` message` issue and it looks like it is due to the `cr` added in https://github.com/acquia/blt/pull/652. In reviewing past closed issues and associated BLT commands / bugfixes, having this `cr` has resulted in some difficult to debug deploy and update issues and in some cases there are workarounds (such as importing config or features twice) that can be refactored in BLT once this is fixed. 

Resolving this is particularly important for ACSF and multisite projects since running a `cr` in the middle of a deploy / update can break task that are intended to be asynchronous. 

This was one of the things fixed in #3006 that was reverted, specifically https://github.com/acquia/blt/pull/3006/commits/a83b8e033d383da5c02cf339b9856095e2a602d0 , https://github.com/acquia/blt/pull/3006/commits/7404038812647586ee12d003ba2f11e2e8227f76, and https://github.com/acquia/blt/pull/3006/commits/d57600fc4e80b7a8eb5caa750e2c67d034d38c2f

In taking a quick look again, we will likely see additional improvements from adding a flush of the service container. Do you want me to create a separate bug to track this bug while the fixes are in progress, or just update this one?

**Comment by danepowell** [comment]
SQLSTATE appears to come from here: https://github.com/acquia/blt/blob/ec4607c480236cc44dec2e0f1eb2e0c01d13b5d0/tests/phpunit/src/BltProjectTestBase.php#L90

The fact that it's not failing the build is expected, since we explicitly allow that command to fail. We must have known it was flaky when we implemented it in https://github.com/acquia/blt/pull/2456

The error is easily fixed by removing the cache rebuild: #3474. This will also do wonders to aid with #3436 (it reduces log output by 15%, or 1500 lines)

To the other points in this thread:
- I can't replicate the file_include errors, they must have been fixed already.
- I think #652 is entirely unrelated, that's not the cache clear causing these errors.
